### PR TITLE
Improve sample migration output

### PR DESCRIPTION
**Pull request by dhruvdutt**
- Improve migration output for sample webpack config.
- Comparison variable parity

**What kind of change does this PR introduce?**
Enhancement

**Summary**
image

Actual output:
image

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by ev1stensberg** [comment]
You need to change your commit format, change you rebase? It should be something like: 

`misc(migrate): prettify output`

**Comment by dhruvdutt** [comment]
@ev1stensberg updated commit messages. :+1:

File: MIGRATE.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ module.exports = {
 
 In summary, we can see the follow changes were made
 1.  The webpack schema for using loaders has changed
-    - `loaders` is now `module.rules`
+    - `module.loaders` is now `module.rules`
```
Comment (ev1stensberg) [inline MIGRATE.md]: In webpack v1 this was loaders, please remove this

File: MIGRATE.md [hunk 2]
```diff
@@ -68,23 +68,23 @@ module.exports = {
 The `migrate` command, when run, will show the proposed changes to the config file in the terminal, prompting the user to
 accept the changes or not:
 
-```bash
+```diff
```
Comment (ev1stensberg) [inline MIGRATE.md]: Looks nice

File: MIGRATE.md (hunk 1 above)
Comment (dhruvdutt) [reply MIGRATE.md]: Cleaned the commit containing that change.

File: MIGRATE.md [hunk 3]
```diff
@@ -68,23 +68,23 @@ module.exports = {
...
-        })
+-    loaders: [
++      rules: [
+-        loader: 'babel',
+-          query: {
++            use: [{
++              loader: 'babel-loader'
++            }],
++            options: {
+-              loader: ExtractTextPlugin.extract('style', 'css!sass')
```
Comment (ev1stensberg) [inline MIGRATE.md]: indent here

Comment (dhruvdutt) [reply MIGRATE.md]: Fixed. Updated the summary image as well. :100:

**Pull request merged**